test/unit/acc/src/AccTagTest.cpp

```diff
@@ -0,0 +1,218 @@
+/* Copyright 2022 Simeon Ehrig
+ *
+ * This file is part of alpaka.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <alpaka/alpaka.hpp>
+#include <alpaka/test/acc/TestAccs.hpp>
+
+#include <catch2/catch_template_test_macros.hpp>
+#include <catch2/catch_test_macros.hpp>
+
+#include <iostream>
+#include <string>
+
+#if 0
+
+using Dim = alpaka::DimInt<1>;
+using Idx = int;
+using TestAccs = alpaka::test::EnabledAccs<Dim, Idx>;
+
+using TagList = std::tuple<
+    alpaka::AccCpuSerialTag,
+    alpaka::AccCpuThreadsTag,
+    alpaka::AccCpuFibersTag,
+    alpaka::AccCpuTbbBlocksTag,
+    alpaka::AccCpuOmp2BlocksTag,
+    alpaka::AccCpuOmp2ThreadsTag,
+    alpaka::AccOmp5Tag,
+    alpaka::AccOaccTag,
+    alpaka::AccGpuCudaRtTag,
+    alpaka::AccGpuHipRtTag,
+    alpaka::AccCpuSyclIntelTag,
+    alpaka::AccFpgaSyclIntelTag,
+    alpaka::AccFpgaSyclXilinxTag,
+    alpaka::AccGpuSyclIntelTag>;
+
+using AccTagTuple = std::tuple<
+    std::tuple<alpaka::test::detail::AccCpuSerialIfAvailableElseInt<Dim, Idx>, alpaka::AccCpuSerialTag>,
```

Comment:
In mp11 jargon, this is a map:
```suggestion
using AccToTagMap = std::tuple<
    std::tuple<alpaka::test::detail::AccCpuSerialIfAvailableElseInt<Dim, Idx>, alpaka::AccCpuSerialTag>,
```